assets/js/blocks/product-gallery/block.json

```diff
@@ -47,5 +48,6 @@
 			"type": "boolean",
 			"default": false
 		}
-	}
+	},
+	"viewScript": "wc-product-gallery-interactivity-view"
```

Comment:
The Product Button named "wc-product-gallery-interactivity-frontend" because currently it has no support for the `view.{js,ts}` file, so basically it is using the `frontend.tsx` file that historically has been utilized by "static" blocks. InteractivityAPI by [default has an "view.js" file](https://github.com/WordPress/gutenberg/blob/trunk/packages/interactivity/docs/2-api-reference.md#the-store) to create and manage the store, so I created a [PR that gives support to that](https://github.com/woocommerce/woocommerce-blocks/pull/10714). That PR should be merged before this PR.

Reply:
You are correct. Thank you for providing the additional context. Ignore this suggestion.